calico-cloud/networking/egress/egress-gateway-azure.mdx

```diff
@@ -0,0 +1,678 @@
+Route Servers. To enable BGP routing protocol, set `bgp` field to `Enabled` in the default installation:
+
+```batch
+kubectl patch installation default --type='merge' -p '{"spec": {"calicoNetwork": {"bgp": "Enabled"}}}'
+```
+
+and then restart the {{prodname}} daemonSet:
+
+```batch
+kubectl rollout restart ds calico-node -n calico-system
+```
+
+### Provision an egress IP pool
+
+Provision a small IP Pool with the range of source IPs that you want to use for a particular
+application when it connects to an external service. For example:
+
+```batch
+kubectl apply -f - <<EOF
+apiVersion: projectcalico.org/v3
+kind: IPPool
+metadata:
+  name: egress-ip-red-pool
+spec:
+  cidr: 10.10.10.0/30
+  blockSize: 31
+  nodeSelector: "!all()"
+  vxlanMode: Never
+EOF
+```
+
+Where:
+- `blockSize` must be specified when the prefix length of the whole `cidr` is more than the default `blockSize` of 26.
+
+- `nodeSelector: "!all()"` is recommended so that this egress IP pool is not accidentally used for cluster pods in general. Specifying this `nodeSelector` means that the IP pool is only used for pods that explicitly identify it in their `cni.projectcalico.org/ipv4pools` annotation.
+
+- Set `vxlanMode` to `Never` since in Azure we rely on Azure fabric to route traffic to egress gateway pods.
+
+### (Optional) Limit number of route advertisements
+
+It is only necessary to advertise the egress IP pool, created in the previous section, to Azure Route Servers. By default {{prodname}}
+advertise all other IP pools, which does not affect any functionality. However, Azure Route Servers have a
+[limit](https://learn.microsoft.com/en-us/azure/route-server/overview#route-server-limits) of 1000 advertisements per peer.
+This limit should not be an issue in a small or medium size cluster, but could be for large size cluster. In order to prevent hitting the limit,
+it is possible to create [BGP filters](../configuring/bgp.mdx#configure-a-bgp-filter) to advertise only egress IP pools. As an example,
+for the IP pool created in the previous section, configure a BGP filter by running the following command:
+
+```batch
+kubectl apply -f - <<EOF
+kind: BGPFilter
+apiVersion: projectcalico.org/v3
+metadata:
+  name: export-egress-ips
+spec:
+  exportV4:
+    - action: Accept
+      matchOperator: In
+      cidr: 10.10.10.0/30
+EOF
+```
+
+Even with BGP filter, {{prodname}} still advertise pod IP blocks. To prevent {{prodname}} advertising pod IP blocks, it is possible to set
+`disableBGPExport` to `true` in the default IP pools.
+
+### Configure route reflectors
+
+{{prodname}} nodes can be configured to act as route reflectors. To do this, each node that you want to act as a route reflector
+must have a cluster ID - typically an unused IPv4 address.
+
+To configure a node to be a route reflector with cluster ID 244.0.0.1, run the following command:
+
+```batch
+calicoctl patch node <node name> -p '{"spec": {"bgp": {"routeReflectorClusterID": "244.0.0.1"}}}'
```

Comment:
This will not work for Calico Cloud, we do not make the calicoctl command available to cloud users.

Reply:
Had a chat with @mazdakn. Looks like ```calicoctl``` is the only way to configure route reflector. Since calicoctl is not available to cloud users, I think EGW on azure won't work for calico cloud. I feel its better to remove the azure doc for calico cloud. WDYT?